### PR TITLE
Sprint 9 logging

### PR DESCRIPTION
**Pull request by Maria-Volkova-Tula**

### CODE REVIEW

File: dev_docs/docs/Fundamentals/Logging/application-insights.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+[Read more and install the VC Application Insights module](https://github.com/VirtoCommerce/vc-module-app-insights){ .md-button }
+
+You can still use MS Azure Application Insights by installing it separately as a module:
+
+* Without extra settings.
+* Setting the required configurations if you prefer a customized version.
+
+## Configuring Application Insights
+
+!!! Note "Important"
```
Comment (tatarincev) [inline dev_docs/docs/Fundamentals/Logging/application-insights.md]: Move this section to how_to_update article

File: dev_docs/docs/Fundamentals/Logging/application-insights.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# MS Azure Application Insights 
+
+Application Insights was integrated to the VC platform till the release of 3.314.0 version, where it was replaced with Serilog library. 
+
+[Read more and install the VC Application Insights module](https://github.com/VirtoCommerce/vc-module-app-insights){ .md-button }
+
+You can still use MS Azure Application Insights by installing it separately as a module:
```
Comment (tatarincev) [inline dev_docs/docs/Fundamentals/Logging/application-insights.md:7]: Add linkto how to update

File: dev_docs/docs/Fundamentals/Logging/overview.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+* Serilog (built-in in the platform). 
+
+    [Read more](https://serilog.net/){ .md-button }
+
+* MS Azure Application Insights (as separated module). 
+
+    [Read more](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview){ .md-button }
+
+!!! NOTE
+
```
Comment (tatarincev) [inline dev_docs/docs/Fundamentals/Logging/overview.md:14]: Add link to how to update article

**Review by tatarincev — CHANGES_REQUESTED** [review]

File: dev_docs/docs/Fundamentals/Logging/overview.md (hunk 3 above)
Comment (Maria-Volkova-Tula) [reply dev_docs/docs/Fundamentals/Logging/overview.md:14]: image
There's one in the references list

**Pull request merged**